gtsam/nonlinear/Values.h

```diff
@@ -397,7 +397,7 @@ namespace gtsam {
     template<class ValueType>
     size_t count() const {
       size_t i = 0;
-      for (const auto& key_value : *this) {
+      for (const auto key_value : *this) {
```

Comment:
No, see, I want (someone) to fix this, but clang says & does not make sense here. Apparently Values iterators don't give back references. It would be nice if they did.